tests/acceptance/10_files/02_maintain/absolute_link_and_no_move_obstructions.cf

```diff
@@ -44,8 +44,8 @@ bundle agent test
 
   files:
       "$(G.testfile)"
-      classes => init_if_failed("test_ok"),
-      link_from => test_link;
+          classes => init_if_failed("test_ok"),
+          link_from => test_link;
```

Comment:
```
      "promiser"
        attribute => "value";
```
The 2 spaces under the promises are almost always 0 spaces in our acceptance tests, which is unreadable. Furthermore, since the promisers first character is a quote, which looks like a space, the first letter of the promiser is way too close to the first letter of the attribute underneath.

Anyway, I just indented to 4 spaces based on how easily I could read that policy the moment I was editing it.